superchain/configs/sepolia/minato.toml

```diff
@@ -0,0 +1,58 @@
+name = "Minato"
+chain_id = 1946
+public_rpc = "https://rpc.minato.soneium.org"
+sequencer_rpc = "https://rpc.minato.soneium.org"
+explorer = "https://soneium-minato.blockscout.com/"
+superchain_level = 0
+standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
+batch_inbox_addr = "0x00Ca81E019a5312d404aE7fe8367D30DE4c11365"
+canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
```

Comment:
while we're following up on the failing test, can you please add your fjord and granite activation times to this file and rerun the `just codegen` command?

Reply:
Done thanks